### ISSUE TITLE
Automatic Scrap Of New Episodes Of Already Identifed TV Shows

### ISSUE THREAD

**Issue by lynxstrike** [Komet/MediaElch#685]
It's a little bit tedious to have to manually scrap the new episodes of each and every one of the already identified tv shows... when easily MediaElch could do it by itself.

Hope some day you could implement it...

Thanks for all your hard work...

**Comment by psonnosp** [comment]
No please don't implement anything that happens automatically.
There are other managers that do stuff like this so please begin using one of those if you have a need for this instead of ruining MediaElch with it.

The thing I like the most with this program is that you have total control of everything that happens, please don't ever change that!

**Comment by ludi113** [comment]
Maybe an option in the settings to enable/disable the automatic scraping would please both users; default set to off to keep the current behaviour.

**Comment by psonnosp** [comment]
As long as that doesn't mean we end up going down that slippery slope and ending up like Ember MM with 5.000 different functions with 50.000 different options on 100 different settings pages.

Keep Mediaelch clean and let the ones that need a program that does everything but bring you coffee in the morning use another program cause there are plenty of them out there.

**Comment by lynxstrike** [comment]
What I proposed is that the "already with TVDB id" shows, with new episodes, we select and in the context menu select the option "scrape" and those shows automatically scrape the info for the new episodes... it would be immensely helpful and you would finish the scraping chores in a flash..

... so, what do you think ...?

**Comment by ludi113** [comment]
Are you looking to bypass the dialog that shows the tv show name to the user? At the bottom of the dialog there is a popup where you can choose what part of the tv show to scrape. You can also scrape the whole tv show in that action. 

However, if you want to automatically scrape several tv shows in one action, that would be a real problem to me, because my tv shows are not all the same language. I use to switch language depending on the tv show before scraping. Once the feature #740 has been implemented, the situation would be different.

**Comment by lynxstrike** [comment]
Each show should be scraped with the same attributes it has (language) and only scrape the new episodes (or all, depending on selection) and should be seamless... 
This is not about scraping the whole show... just the new episodes one is adding ...
Any way, this is my suggestion... maybe there is a better alternative to do this chore more easily....

**Comment by psonnosp** [comment]
Yes the alternative is to use a program that already has the function you're after. Like Ember MM that scrape stuff as fast as you think about doing it and don't ask a single question.

You might think I sound like a dick bashing your idea but I'm so incredible happy that there still are a clean, efficient and simple program like MediaElch around that makes it easy to control every step and I really, really want it to stay like that.
There are several more or less automated media managers with hundreds of options and functions that do what you want so I don't see a point in turning MediaElch into another one of those.
Even if this is a tiny thing it might be the thing that set the ball rolling...

**Comment by bugwelle** [comment]
@lynxstrike I see that you have closed this issue. Just wanted to let you know that I'm re-writing the whole TV scraper interfaces.  This may be very well possible in future versions. :)

**Comment by lynxstrike** [comment]
@bugwelle 

Thanks for keeping an eye in this issue. I closed it to help reduce the clotter and many thanks for your comments. I'm reopening so if you decide to implement it in the future helps you ...

I think it would be a great improvement to MediaElch (it takes me nearly 100 min to scrape new episodes into my tv shows collection while with another tool it takes 5 min and without my intervention). I think it would be game changer to have this option (keyword "option"... for those of us who like a little automation with a neatly organized collection).

Any way, I leave it to your discretion, and thanks for keeping this great software free, up to date and one of the best!

**Comment by bugwelle** [comment]
Thanks. :-)
